### PR TITLE
First structure

### PR DESCRIPTION
**Pull request by GeorgesFORTRY**

### CODE REVIEW

File: src/components/Header.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.Header {
+  padding-top: 30px;
+  background-color: black;
+  color : white;
+
```
Comment (JulienDelemarle) [inline src/components/Header.scss:5]: be aware to avoid empty lines

File: src/components/Main.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    .intro {
+        width: 50vw;
+        padding: 120px;
+        p {     
+        font-size: 50px;
+        text-align: center;
+        padding-bottom: 50px;
+        }
+
+
```
Comment (JulienDelemarle) [inline src/components/Main.scss:16]: be aware to avoid empty lines

**Review by JulienDelemarle — APPROVED** [review]
changes ok. 
for next commits, delete empty lines in css

**Review by LoicBrassart — APPROVED** [review]
Some indentation problems, need to fix it quickkly for the custom project template

**Pull request merged**